### DISCUSSION TITLE
adding button to audio-slider plugin

### DISCUSSION THREAD

**Discussion by EceKayaMPI** [jspsych/jsPsych#2426] [Q&A] [answered]
Hi, 
I wish to add a button in audio-slider plugin, which will replay audio. Is there a way to do this? I am using jspsych version 7.1.

Thanks in advance.

- Ece

**Comment by EceKayaMPI** [comment, marked as answer]
https://github.com/jspsych/jsPsych/discussions/595#discussioncomment-1940424

**Reply by GABowers** [reply]
That thread is about the `audio-button-response` plugin. Did you manage to get a scale/slider working with an extra button somehow?

**Comment by ineminor** [comment]
Have you solved this problem yet?
